packages/manager/modules/dbaas-logs/src/logs/order/logs-order.controller.js

```diff
@@ -0,0 +1,142 @@
+import grafana from '../../assets/grafana.png';
+import graylog from '../../assets/graylog.png';
+import kibana from '../../assets/kibana.png';
+
+export default class LogsOrderCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $http,
+    $state,
+    LogsConstants,
+    CucOrderHelperService,
+    ovhDocUrl,
+    coreConfig,
+    $translate,
+    atInternet,
+  ) {
+    this.$http = $http;
+    this.$q = $q;
+    this.$translate = $translate;
+    this.$state = $state;
+    this.atInternet = atInternet;
+    this.region = coreConfig.getRegion();
+    this.LogsConstants = LogsConstants;
+    this.CucOrderHelperService = CucOrderHelperService;
+    this.ovhDocUrl = ovhDocUrl;
+    this.urls = {};
+
+    this.assets = {
+      grafana,
+      graylog,
+      kibana,
+    };
+  }
+
+  $onInit() {
+    this.isLoading = true;
+    this.orderError = false;
+    this.orderSucceed = false;
+    this.selectedOffer = undefined;
+    this.selectedDatacenter = undefined;
+    this.workflowOptions = {
+      catalog: undefined,
+      catalogItemTypeName: 'plans',
+      productName: 'logs',
+      getPlanCode: () => {
+        return this.selectedOffer;
+      },
+      onGetConfiguration: () => {
+        return [
+          {
+            label: 'region',
+            value: this.selectedDatacenter,
+          },
+        ];
+      },
+    };
+    this.fetchOffer();
+
+    this.urls.docsUrl = this.ovhDocUrl.getDocUrl(
+      this.LogsConstants.LOGS_DOCS_NAME,
+    );
+    this.CucOrderHelperService.buildUrl(
+      this.LogsConstants.LOGS_PRODUCT_URL,
+    ).then((url) => {
+      this.urls.productURL = url;
+    });
+    this.CucOrderHelperService.buildUrl(this.LogsConstants.ORDER_URL).then(
+      (url) => {
+        this.urls.orderURL = url;
+      },
+    );
+
+    this.atInternet.trackClick({
+      name: `cloud::dbaas::logs::create-account`,
+      type: 'clic',
+    });
+  }
+
+  selectOffer(offerName) {
+    this.selectedOffer = offerName;
+  }
+
+  selectDatacenter(datacenter) {
+    this.selectedDatacenter = datacenter;
+  }
+
+  fetchOffer() {
+    return this.$http
+      .get('/order/catalog/public/logs', {
+        params: {
+          ovhSubsidiary: this.user.ovhSubsidiary,
+        },
+      })
+      .then(({ data }) => {
+        this.workflowOptions.catalog = data;
+        this.isLoading = false;
+      });
+  }
+
+  onOrderSucceed() {
+    this.atInternet.trackClick({
+      name: `cloud::dbaas::logs::create-account::confirm`,
+      type: 'clic',
```

Comment:
i'm not sure type 'clic' is valid

Reply:
on excel, i have "type de hint" to "clic". i have seen "actions" on some other part of code, i can change if needed

Reply:
@Heiiji I'm facing issue understanding your comment. For available types on tracking you can refer to https://github.com/ovh/manager/tree/master/packages/components/ng-at-internet